src/metabase/util/malli/fn.clj

```diff
@@ -150,7 +150,9 @@
 (defn- validate [error-context schema value error-type]
   (when *enforce*
     (when-let [error (mr/explain schema value)]
-      (let [humanized (me/humanize error)
+      (let [humanized (me/humanize error {:wrap (core/fn humanize-include-value
+                                                  [{:keys [value message]}]
+                                                  (str message ", got: " (pr-str value)))})
```

Comment:
decided to go with `got` so that it's clear where errors are coming from. There's a similar thing for `defendpoint` that says received.